Force waitress to use polling instead of select. This fixes the following
error:
    ValueError: filedescriptor out of range in select()

see https://github.com/Pylons/waitress/issues/31#issuecomment-480894686
--- anki-2.1.49/qt/aqt/mediasrv.py
+++ anki-2.1.49_asyncore_use_poll/qt/aqt/mediasrv.py
@@ -77,6 +77,7 @@
                 host=desired_host,
                 port=desired_port,
                 clear_untrusted_proxy_headers=True,
+                asyncore_use_poll=True,
             )
             if devMode:
                 print(
